### PR TITLE
[Sprint 7][Task #144475] Feature manage request list UI

### PR DESCRIPTION
**Pull request by daolq3012**
- Detail: manage request list UI
- Redmine: https://dev.framgia.com/redmine/issues/144475
- Effect: None

### CODE REVIEW

File: src/wsm/wsm/viewcontrollers/ManageRequestListViewController.swift [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView?.register(UINib(nibName: "ManageRequestTableViewCell", bundle: nil), forCellReuseIdentifier: "ManageRequestTableViewCell")
+    }
+}
+
+extension ManageRequestListViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "ManageRequestTableViewCell")
```
Comment (duongdang152) [inline src/wsm/wsm/viewcontrollers/ManageRequestListViewController.swift]: đang mock view ah

Comment (daolq3012) [reply src/wsm/wsm/viewcontrollers/ManageRequestListViewController.swift]: Đâu, t khai báo file xib rồi lấy ra dùng ấy chứ

Comment (daolq3012) [reply src/wsm/wsm/viewcontrollers/ManageRequestListViewController.swift]: móc view

File: src/wsm/wsm/viewcontrollers/RequestType.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+//
+//  Created by DaoLQ on 10/30/17.
+//  Copyright © 2017 framgia. All rights reserved.
+//
+
+import Foundation
+import InAppLocalize
+
+enum RequestType {
+    case Others
```
Comment (LeVanTuan) [inline src/wsm/wsm/viewcontrollers/RequestType.swift]: lowcase

File: src/wsm/wsm/viewcontrollers/ManageRequestListViewController.swift [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+//  wsm
+//
+//  Created by DaoLQ on 10/26/17.
+//  Copyright © 2017 framgia. All rights reserved.
+//
+
+import UIKit
+import InAppLocalize
+
+class ManageRequestListViewController: UIViewController {
```
Comment (nguyennls) [inline src/wsm/wsm/viewcontrollers/ManageRequestListViewController.swift]: răng k dùng baseviewcontroller

Comment (daolq3012) [reply src/wsm/wsm/viewcontrollers/ManageRequestListViewController.swift]: fixed

**Pull request merged**